proto/oracle/v1/oracle.proto

```diff
@@ -122,16 +122,9 @@ message ExchangeRateTuple {
   ];
 }
 
-// PairReward defines a credit object towards validators
+// Rewards defines a credit object towards validators
 // which provide prices faithfully for different pairs.
-message PairReward {
-  // pair defines the pair for which we incentivize validator to provide prices
-  // for.
-  string pair = 1 [
-    (gogoproto.customtype) =
-        "github.com/NibiruChain/nibiru/x/common/asset.Pair",
-    (gogoproto.nullable) = false
-  ];
+message Rewards {
   // id uniquely identifies the rewards instance of the pair
   uint64 id = 2;
```

Comment:
Maybe shift the field numbers back to 1?

Reply:
it was out of proto habit! :D